Mathlib/Algebra/Polynomial/EraseLead.lean

```diff
@@ -173,6 +173,9 @@ theorem eraseLead_add_of_natDegree_lt_right {p q : R[X]} (pq : p.natDegree < q.n
 theorem eraseLead_degree_le : (eraseLead f).degree ≤ f.degree :=
   f.degree_erase_le _
 
+theorem eraseLead_degree_lt (hf : f ≠ 0) : (eraseLead f).degree < f.degree :=
```

Comment:
I see that the naming convention is also direspected in the rest of the file, but this really should be
```suggestion
theorem degree_eraseLead_lt (hf : f ≠ 0) : (eraseLead f).degree < f.degree :=
```